mysql-cli.sh

```diff
@@ -1,6 +1,6 @@
 #! /bin/bash -e
 
-docker run ${1:--it} \
+docker run $* \
```

Comment:
Having it default to -it is useful for humans.
is it possible to restore this behavior. i.e. `if -z "$*" ; then ....`???